### PR TITLE
linuxPackages.acpi_call: 2020-04-07 -> 1.2.1

### PR DESCRIPTION
**Pull request by dotlambda**
###### Motivation for this change
Use a version that other distros use too: https://repology.org/project/acpi-call

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116410` at 9caa5abe run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>linuxPackages-libre.acpi_call</li>
    <li>linuxPackages_hardkernel_4_14.acpi_call</li>
    <li>linuxPackages_hardkernel_latest.acpi_call</li>
    <li>linuxPackages_latest-libre.acpi_call</li>
  </ul>
</details>
<details>
  <summary>16 packages built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/709b5c0d14117cc8af6232f8d49ba284">linuxPackages.acpi_call</a></li> (linuxPackages_5_4.acpi_call)</li>
    <li><a href="https://gist.github.com/379d0072d38d7d4f5f49ccaf40d242dc">linuxPackages_4_14.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/e8ebf0b5c7f68062574778f481cc6ee3">linuxPackages_4_19.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/e91f9b5d30cc1a6058017a59fcb92b0f">linuxPackages_4_4.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/1808818a0cfaa212df0b84714bc864b6">linuxPackages_4_9.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/fde985aad2acb7e32ef10bca24d5f4de">linuxPackages_5_10.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/5e70695e69e92b0ccbda1b39804a33b0">linuxPackages_5_11.acpi_call</a></li> (linuxPackages_latest.acpi_call)</li>
    <li><a href="https://gist.github.com/31b617a32a240367c5901118a5600176">linuxPackages_hardened.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/a971935c1cf9e134b37ef213b1587e91">linuxPackages_latest_hardened.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/dfa2dcfd3b4463c5df12e3fac115fa8b">linuxPackages_latest_xen_dom0.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/a529881a4efc137dde5570498f503091">linuxPackages_latest_xen_dom0_hardened.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/4d4b0bd8ef1589d332001bce3eeb1554">linuxPackages_lqx.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/bfbce767bd0104ddf4a49e55df2aadf7">linuxPackages_testing_bcachefs.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/09ca1f2e6ac3ef98d668afe62ac8d167">linuxPackages_xen_dom0.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/1a6e368c97b977646067ed5cd8294b84">linuxPackages_xen_dom0_hardened.acpi_call</a></li></li>
    <li><a href="https://gist.github.com/dcb55fa017fd10237aed4d6704e0e7de">linuxPackages_zen.acpi_call</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/os-specific/linux/acpi-call/default.nix:23:3:
```
   |
23 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/os-specific/linux/acpi-call/default.nix [hunk 1]
```diff
@@ -29,5 +30,6 @@ stdenv.mkDerivation rec {
     inherit (src.meta) homepage;
```
Comment (SuperSandro2000) [inline pkgs/os-specific/linux/acpi-call/default.nix:30]: Please set the homepage explicitly.

Comment (dotlambda) [reply pkgs/os-specific/linux/acpi-call/default.nix:30]: Come on, no need to change random stuff in the expression.

**Comment by Mic92** [comment]
Result of `nixpkgs-review pr 116410` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages marked as broken and skipped:</summary>
  <ul>
    <li>linuxPackages-libre.acpi_call</li>
    <li>linuxPackages_hardkernel_4_14.acpi_call</li>
    <li>linuxPackages_hardkernel_latest.acpi_call</li>
    <li>linuxPackages_latest-libre.acpi_call</li>
  </ul>
</details>
<details>
  <summary>16 packages built:</summary>
  <ul>
    <li>linuxPackages.acpi_call (linuxPackages_5_4.acpi_call)</li>
    <li>linuxPackages_4_14.acpi_call</li>
    <li>linuxPackages_4_19.acpi_call</li>
    <li>linuxPackages_4_4.acpi_call</li>
    <li>linuxPackages_4_9.acpi_call</li>
    <li>linuxPackages_5_10.acpi_call</li>
    <li>linuxPackages_5_11.acpi_call (linuxPackages_latest.acpi_call)</li>
    <li>linuxPackages_hardened.acpi_call</li>
    <li>linuxPackages_latest_hardened.acpi_call</li>
    <li>linuxPackages_latest_xen_dom0.acpi_call</li>
    <li>linuxPackages_latest_xen_dom0_hardened.acpi_call</li>
    <li>linuxPackages_lqx.acpi_call</li>
    <li>linuxPackages_testing_bcachefs.acpi_call</li>
    <li>linuxPackages_xen_dom0.acpi_call</li>
    <li>linuxPackages_xen_dom0_hardened.acpi_call</li>
    <li>linuxPackages_zen.acpi_call</li>
  </ul>
</details>

**Pull request merged**